apps/frontend/src/modules/auth/slices/actions.ts

```diff
@@ -63,6 +64,14 @@ const getAuthenticatedUser = createAsyncThunk<
 		return null;
 	}
 
+	const { exp } = jwtDecode(token as string);
```

Comment:
Should not we do this check on backend?
The flow:
1. User makes request to backend.
2. Backend checks jwt expiration date
3. Backend sends the expiration error on client

Reply:
I am afraid that the frontend should know nothing about decoding jwt token. It will be safe to do it on backend
@fshabanov what do you think?

Reply:
Agree. The only thing we need to do is to remove the token if `getAuthenticatedUser` fails